algorithms/kernel/dtrees/forest/classification/df_classification_predict_dense_default_batch_impl.i

```diff
@@ -590,13 +615,8 @@ Status PredictClassificationTask<algorithmFPType, cpu>::predictAllPointsByAllTre
 
             for (size_t iRes = 0; iRes < nRowsToProcess; ++iRes)
             {
-                PRAGMA_IVDEP
-                PRAGMA_VECTOR_ALWAYS
-                for (size_t j = 0; j < _nClasses; j++)
-                {
-                    prob_internal[iRes * _nClasses + j] = algorithmFPType(prob_internal[iRes * _nClasses + j]) * inverseNTreesTotal;
-                }
-                if (_res)
+                normalizeProbabilities(prob_internal + iRes * _nClasses);
```

Comment:
I think you can merge normalizeProbabilities and getMaxClass at once to speedup computations.

Reply:
+1